documentation/src/docs/getting-started.md

```diff
@@ -0,0 +1,103 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega Flink Connectors [![Build Status](https://travis-ci.org/pravega/flink-connectors.svg?branch=master)](https://travis-ci.org/pravega/flink-connectors)
+
+Connectors to read and write [Pravega](http://pravega.io/) Streams with [Apache Flink](http://flink.apache.org/) stream processing framework.
+
+Build end-to-end stream processing pipelines that use Pravega as the stream storage and message bus, and Apache Flink for computation over the streams.
+
+
+## Features & Highlights
+
+  - **Exactly-once processing guarantees** for both Reader and Writer, supporting **end-to-end exactly-once processing pipelines**
+
+  - Seamless integration with Flink's checkpoints and savepoints.
+
+  - Parallel Readers and Writers supporting high throughput and low latency processing.

```

Comment:
"high throughput and low latency" doesn't sound necessary in this getting-started markdown.
Instead we may emphasis that Readers/Writers are themselves in parellel, and in particular, Readers/Writers are independent in terms of performance as they operate on different parts of stream?